src/events/order/cd-event-sessions.vue

```diff
@@ -124,6 +123,7 @@
         }
         return this.existingChildren;
       },
+      ...mapGetters('order', ['event']),
```

Comment:
Huh... I thought we had to set the `store` on each component, but it seems like it inherits from the parent... 👍